### DISCUSSION TITLE
Contents of UI kit

### DISCUSSION THREAD

**Discussion by IamSanjana** [GBKS/bitcoin-wallet-ui-kit#8] [General]
Is there any documentation on prospects of the UI contents of the kit? I might have some rough ideas to discuss on what else can be included.

**Comment by GBKS** [comment]
By prospects, you mean future plans?.

There are multiple aspects to this project:
1. Basic elements (colors, type styles, layout parameters, etc)
1. Basic components (buttons, inputs, etc)
1. Bitcoin-specific components (balance display, fee selector, etc)
1. Screen & user-flow mock-ups

1-3 are a general toolbox for designers to build anything they want. 4 closely tracks designs used in the [Bitcoin Design Guide](https://bitcoin.design/guide/).

Beyond the Figma file itself, there are some efforts to implement the kit as code libraries (see [here](https://github.com/reez/WalletUI) and [here](https://github.com/Aman035/wallet-ui-react)). Would be amazing to see those projects grow further.

The main way I've gone about evolving the UI kit so far has been through work on the guide and also observing how others use it, and implementing those learnings. 

Those are just my thoughts at the moment, hope it helps. What would you like to see added?
